learning_journal/templates/login.jinja2

```diff
@@ -0,0 +1,21 @@
+
+{% extends "layout.jinja2" %}
+    {% block title %}
+        Edit Entry
+    {% endblock %}
+    {% block body %}
+    <div class="well">
+        <h4>Login:</h4>
+        <form role="form" method="POST">
+            <input type="hidden" name="csrf_token" value="{{ request.session.get_csrf_token() }}" />
+            <div class="form-group">
+                <p>UserName:</p>
+                <input class="form-control" name="username"></input>
+                <br>
+                <p>Password<p>
+                <textarea class="form-control" name="password" rows="10"></textarea>
```

Comment:
make this a password field so it is not displayed on the screen.  Otherwise, a spy hiding in a small, fake shrub behind you will get access to your super-secret information!!